test/build/dotenv-webpack-plugin/dotenv-webpack-plugin.test.js

```diff
@@ -0,0 +1,74 @@
+"use strict";
+
+const { run, readFile } = require("../../utils/test-utils");
+const { resolve } = require("path");
+const { existsSync } = require("fs");
+
+const assertNoErrors = (exitCode, stderr, stdout, testDir) => {
+  expect(exitCode).toBe(0);
+  expect(stderr).toBeFalsy();
+  expect(stdout).toBeTruthy();
+  expect(existsSync(resolve(testDir, "./dist/main.js"))).toBeTruthy();
+};
+
+const getBuildOutput = async (testDir) => {
+  try {
+    return readFile(resolve(testDir, "./dist/main.js"), "utf-8");
+  } catch (error) {
+    expect(error).toBe(null);
+  }
+};
+
+describe("dotenv-webpack-plugin", () => {
+  it("reads .env file and defines variables correctly", async () => {
+    const testDir = __dirname + "/builtin-config";
```

Comment:
we should use `path.resolve` here

Reply:
using `path.join` since we are using `__dirname`